packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreAliased.spec.ts

```diff
@@ -4,99 +4,50 @@
  */
 
 import { strict as assert } from "assert";
-import {
-    ContainerRuntimeFactoryWithDefaultDataStore,
-    DataObjectFactory,
-} from "@fluidframework/aqueduct";
 import { IContainer } from "@fluidframework/container-definitions";
-import { IRequest } from "@fluidframework/core-interfaces";
+import { ContainerRuntime } from "@fluidframework/container-runtime";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
 import { requestFluidObject } from "@fluidframework/runtime-utils";
 import { ITestObjectProvider } from "@fluidframework/test-utils";
-import { describeNoCompat } from "@fluidframework/test-version-utils";
-import {
-    IContainerRuntimeOptions,
-} from "@fluidframework/container-runtime";
-import { IContainerRuntimeBase } from "@fluidframework/runtime-definitions";
-import { getGCStateFromSummary, TestDataObject } from "../mockSummarizerClient";
-import { mockConfigProvider } from "./mockConfigProivder";
+import { describeNoCompat, ITestDataObject, TestDataObjectType } from "@fluidframework/test-version-utils";
+import { defaultGCConfig } from "./gcTestConfigs";
+import { getGCStateFromSummary, waitForContainerConnection } from "./gcTestSummaryUtils";
 
 /**
  * Validates this scenario: When a datastore is aliased that it is considered a root datastore and always referenced
  */
 describeNoCompat("GC Data Store Aliased", (getTestObjectProvider) => {
     let provider: ITestObjectProvider;
-    const dataObjectFactory = new DataObjectFactory(
-        "TestDataObject",
-        TestDataObject,
-        [],
-        []);
-
-    const runtimeOptions: IContainerRuntimeOptions = {
-        summaryOptions: {
-            summaryConfigOverrides: {
-                state: "disabled",
-            },
-        },
-        gcOptions: {
-            gcAllowed: true,
-        },
-    };
-    const innerRequestHandler = async (request: IRequest, runtime: IContainerRuntimeBase) =>
-        runtime.IFluidHandleContext.resolveHandle(request);
-    const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore(
-        dataObjectFactory,
-        [
-            [dataObjectFactory.type, Promise.resolve(dataObjectFactory)],
-        ],
-        undefined,
-        [innerRequestHandler],
-        runtimeOptions,
-    );
-
-    // Enable config provider setting to write GC data at the root.
-    const settings = { "Fluid.GarbageCollection.LogUnknownOutboundReferences": "true" };
```

Comment:
I noticed this setting being deleted, when do we add it back in? This setting should be on specifically for this test as this test should not have a specific log, and it won't because the setting is off.

Reply:
This is enabled by default now.